recipes/libgd/all/test_package/CMakeLists.txt

```
@@ -1,8 +1,10 @@
 cmake_minimum_required(VERSION 3.1)
-project(libgd_test CXX)
+project(test_package C)
 
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup()
+conan_basic_setup(TARGETS)
 
-add_executable(${PROJECT_NAME} test_package.cpp)
-target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
+find_package(libgd REQUIRED)
```

```suggestion
find_package(libgd CONFIG REQUIRED)
```

cmake_find_package_multi generates Cmake config file only

---

It doesn't really matter.
There is:
```cmake
find_package(libgd CONFIG REQUIRED)  # looks for libgd-config.cmake
find_package(libgd MODULE REQUIRED)  # looks for Findlibgd.cmake
find_package(libgd REQUIRED)  # looks for both Findlibgd.cmake and libgd-config
```